deepmd/utils/tabulate.py

```diff
@@ -53,27 +56,30 @@ def __init__(self,
         except Exception:
             self.sel_a = self.graph.get_operation_by_name('DescrptSeA').get_attr('sel_a')
             self.descrpt = self.graph.get_operation_by_name ('DescrptSeA')
-        self.ntypes = self._get_tensor_value(self.graph.get_tensor_by_name ('descrpt_attr/ntypes:0'))
-        self.davg = self._get_tensor_value(self.graph.get_tensor_by_name ('descrpt_attr/t_avg:0'))
-        self.dstd = self._get_tensor_value(self.graph.get_tensor_by_name ('descrpt_attr/t_std:0'))
+
+        self.davg = get_tensor_by_name(self.model_file, 'descrpt_attr/t_avg')
+        self.dstd = get_tensor_by_name(self.model_file, 'descrpt_attr/t_std')
+        self.ntypes = get_tensor_by_name(self.model_file, 'descrpt_attr/ntypes')
 
         
         self.rcut = self.descrpt.get_attr('rcut_r')
         self.rcut_smth = self.descrpt.get_attr('rcut_r_smth')
 
-        self.filter_variable_nodes = self._load_matrix_node()
+        self.fitting_net_nodes = get_fitting_net_nodes(self.model_file)
+        self.embedding_net_nodes = get_embedding_net_nodes(self.model_file)
```

Comment:
Read file too many times!

Reply:
Have `self.fitting_net_nodes ` and `self.embedding_net_nodes` been used anywhere?